src/component/atom/cardCarousel/cardCarouselCustomize.tsx

```diff
@@ -0,0 +1,16 @@
+import { CardContent, Typography } from "@mui/material";
```

Comment:
What this `atom` directory for?
I think that /src/components/card-carousel is enogh forr keeping all these files for now.

Reply:
The name CardCarousel.styled matches better for this file name.